packages/react/src/analytics/integrations/Vitorino/constants.js

```diff
@@ -17,10 +23,7 @@ export const PROD_SCRIPT_SRC =
 /**
  * Environment types to be passed in to Vitorino.
  */
-export const ENVIRONMENT_TYPES = {
-  prod: 'production',
-  dev: 'development',
-};
+export const ENVIRONMENT_TYPES = AnalyticsConstants;
```

Comment:
If the `ENVIRONMENT_TYPES` are exactly the same as what's already exported on the `AnalyticsConstants`, I don't think we need this extra export (`ENVIRONMENT_TYPES`). You can use directly the `AnalyticsConstants` wherever you need.

Reply:
I use this approach to avoid any breaking changes, so If the tenant are using this EnvironmentTypes, could use as well without any changes.